.github/workflows/ci.yml

```diff
@@ -68,6 +68,13 @@ jobs:
           command: clippy
           args: --all-targets --features shared-memory -- -D warnings
 
+      - name: Clippy transport_shm
+        if: ${{ matrix.os == 'ubuntu-latest' || matrix.os == 'macOS-latest' }}
+        uses: actions-rs/cargo@v1
+        with:
+          command: clippy
+          args: --all-targets --features shared-memory --features transport_shm -- -D warnings
```

Comment:
I would just add `--features transport_shm` to the previous test. I would avoid duplicating the clippy check.